src/main/java/de/retest/web/selenium/UnbreakableDriver.java

```diff
@@ -265,4 +266,12 @@ public void perform( final Collection<Sequence> actions ) {
 	public void resetInputState() {
 		wrapped.resetInputState();
 	}
+
+	@Override
+	public WebDriver getWrappedDriver() {
```

Comment:
Due to the `@Getter` and `@Setter` annotations, we now have `getWrapped` and `getWrappedDriver`. I would suggest to rename the field to `wrappedDriver`, then your getter here overwrites the getter generated by Lombok. Afterwards, you can also adapt the usage in `ScreenshotProvider` and elsewhere.

Reply:
Done. Please re-review.